### PR TITLE
Enable filtering with multiple languages and spoken-languages

### PR DESCRIPTION
**Pull request by fgsepter**
**Checklist**

- [x] My code is properly formatted using the latest [black](https://github.com/psf/black) 
- [x] I have added/updated [tests](https://github.com/hedythedev/starcli/tree/main/tests) if needed 
- [x] I have tried running my code manually 
- [x] I have checked for spelling errors 

**Description**

I have made changes to enable filtering with multiple languages and spoken languages. 
We can search based on various combinations of languages and spoken languages. For example, we can use -l Python+JavaScript to filter two languages Python and JavaScript, and -S zh+en to filter Chinese and English.

### CODE REVIEW

File: starcli/__main__.py [hunk 1]
```diff
@@ -24,7 +24,14 @@
 
 
 @click.command()
-@click.option("--lang", "-l", type=str, default="", help="Language filter eg: python")
+@click.option(
```
Comment (hedyhli) [inline starcli/__main__.py:27]: Let's have this option (and spoken-language) able to be used multiple times, like `-l vim -l python`. Check out how it's done in `--topic`

File: starcli/__main__.py [hunk 2]
```diff
@@ -24,7 +24,14 @@
...
+    "--lang", 
+    "-l", 
+    type=str, 
+    default="", 
+    help="Language filter eg: python"
```
Comment (hedyhli) [inline starcli/__main__.py:32]: Perhaps have this feature reflected in the help text too? (Like in `--topic`)

**Review by hedyhli — CHANGES_REQUESTED** [review]
Haven't tested yet, but looks pretty good so far :)

File: starcli/__main__.py (hunk 2 above)
Comment (fgsepter) [reply starcli/__main__.py:32]: Get it, I will work on this

File: starcli/__main__.py (hunk 1 above)
Comment (fgsepter) [reply starcli/__main__.py:27]: I have modified the ```--lang``` and ```--spoken-language``` and they can be used multiple times now.

**Comment by hedyhli** [comment]
Closing this PR as it's stale. If you have other improvements to the code, feel free to open another PR :)

**Pull request closed without merging**